apps/overige-object-api/src/server.ts

```diff
@@ -0,0 +1,134 @@
+/* eslint-disable no-undef */
+import type { CorsOptions } from 'cors';
+import cors from 'cors';
+import { config } from 'dotenv';
+import express from 'express';
+import { NextFunction, Request, Response } from 'express';
+import * as OpenApiValidator from 'express-openapi-validator';
+import yaml from 'js-yaml';
+import fs from 'node:fs';
+import path from 'node:path';
+import swaggerUi from 'swagger-ui-express';
+import { objects, objecttypes, openapi } from './routers';
+import { envAvailability, ErrorHandler } from './utils';
+config();
+
+type OpenOpenApiValidationError = {
+  path: string;
+  message: string;
+  errorCode: string;
+};
+interface OpenApiValidationErrorTypes {
+  errors: OpenOpenApiValidationError[];
+}
+
+// Validate environment variables
+envAvailability({
+  env: process.env,
+  keys: ['STRAPI_PRIVATE_URL', 'OVERIGE_OBJECT_API_PORT'],
+});
+
+const swaggerDocument: any = yaml.load(fs.readFileSync(path.join(__dirname, './docs/openapi.yaml'), 'utf8'));
+const whitelist = process.env.OVERIGE_OBJECT_API_CORS?.split(', ') || [];
```

Comment:
Op deze manier weet typescript beter dat het een `string[]` is.

```suggestion
const whitelist = (process.env.OVERIGE_OBJECT_API_CORS || '').split(', ');
```